src/plugins/discover/public/application/main/components/layout/use_discover_histogram.ts

```diff
@@ -62,6 +62,7 @@ export const useDiscoverHistogram = ({
 }: UseDiscoverHistogramProps) => {
   const services = useDiscoverServices();
   const savedSearchData$ = stateContainer.dataState.data$;
+  const savedSearchHits$ = savedSearchData$.totalHits$;
```

Comment:
Maybe just `totalHits$`?

Reply:
I think then we should also do this with `savedSearchData$`, wdyt?